wasm/Cargo.toml

```
@@ -16,7 +16,8 @@ members = [
 [profile.dev]
 panic = "abort"
 
-[profile.release]
+[profile.deploy]
+inherits = "release"
```

```suggestion
[profile.deploy]
inherits = "release"
strip = "debuginfo" # Remove debugging info from the binary

[profile.release]
```

I think we want it like so

---

and also move `lto` into `deploy`

---

also move `codegen-units` into `deploy`